src/gen9_vp9_encoder.c

```diff
@@ -1076,6 +1076,7 @@ void gen9_vp9_set_curbe_brc(VADriverContextP ctx,
 
         *param->pbrc_init_reset_buf_size_in_bits  = cmd->dw6.buf_size;
         *param->pbrc_init_reset_input_bits_per_frame  = dInputBitsPerFrame;
+        *param->pbrc_init_current_target_buf_full_in_bits = cmd->dw6.buf_size >> 1;
```

Comment:
It will be better only to initialize it for init scenario instead of both init and reset scenario.